docs/source/services/key_value_services/index.rst

```diff
@@ -19,9 +19,9 @@ This matrix details the current state of available Key Value Services supported
 
     *    - :ref:`Cassandra KVS <cassandra-configuration>`
          - Supported
-         - 2.0+
+         - 2.1.15+, 2.2.5+, 3.0.3+, 3.2+
```

Comment:
has anyone ever actually tested atlas against 3.X? feels weird to declare support for it.
Practically speaking we only support 2.2.9+